correlator.go

```diff
@@ -97,10 +92,21 @@ func handleGet(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(claimChecks[key]))
 }
 
-func sendToChannel(channel string, correlationID string, r *http.Request) error {
-	fmt.Printf("SENDING TO %s\n", channel)
-	// TODO: namespace
-	url := fmt.Sprintf("http://%s-channel.default.svc.cluster.local", channel)
+func parsePathToChannel(path string) (namespace, name string) {
+	s := strings.Split(path, "/")
+	name = s[1]
+	if len(s) >= 3 {
+		namespace = s[2]
+	}
+	if namespace == "" {
+		namespace = "default"
+	}
+	return
+}
+
+func sendToChannel(namespace, channel string, correlationID string, r *http.Request) error {
+	fmt.Printf("SENDING TO %s/%s\n", namespace, channel)
+	url := fmt.Sprintf("http://%s-channel.%s.svc.cluster.local", namespace, channel)
```

Comment:
Shouldn't the args be `channel, namespace`?

Reply:
yep, I had a local commit with the fix that I failed to push up